pkg/scheduler/framework/interfaces/types_test.go

```diff
@@ -149,3 +149,48 @@ func TestTargetClusters_Merge(t *testing.T) {
 		})
 	}
 }
+
+func TestTargetClusters_Merge2(t *testing.T) {
```

Comment:
Please merge this test case to `TestTargetClusters_Merge`.

Reply:
done